exonum-java-binding/cryptocurrency-demo/Readme.md

```diff
@@ -24,6 +24,7 @@ Be sure you installed necessary packages:
 - [git](https://git-scm.com/downloads)
 - [Node.js with npm](https://nodejs.org/en/download/)
 - The [system dependencies](https://exonum.com/doc/version/0.10/get-started/install/) of Exonum. You do _not_ need to manually fetch and compile Exonum.
+- You should have `Exonum Java App` installed and configured. The `exonum-java` binary should be availble via `PATH` environment variable.
```

Comment:
Do we have any online doc about the app installation we could mention here?

Reply:
It is WIP, therefore I'd add a (blocked) task to link the docs

Reply:
Done: https://jira.bf.local/browse/ECR-3143